src/snowflake/snowpark/_internal/ast_utils.py

```diff
@@ -9,6 +9,7 @@
 import os
```

Comment:
can you update with latest [ls-SNOW-1491199-merge-phase0-server-side](https://github.com/snowflakedb/snowpark-python/tree/ls-SNOW-1491199-merge-phase0-server-side) changes?